sonic_installer/bootloader/aboot.py

```diff
@@ -2,13 +2,17 @@
 Bootloader implementation for Aboot used on Arista devices
 """
 
+import base64
 import collections
 import os
 import re
 import subprocess
+import zipfile
 
 import click
 
+from M2Crypto import X509
```

Comment:
>M2Crypto [](start = 5, length = 8)

Add this new dependency in setup.py?

Reply:
It will be add in build_debian.sh in sonic-buildimage.

Reply:
Your statement is true. But by best practice, you need to add new dependency in setup.py

---
In reply to: [446767082](https://github.com/Azure/sonic-utilities/pull/969#discussion_r446767082) [](ancestors = 446767082)

Reply:
Added.